packages/valory/contracts/service_registry/contract.py

```diff
@@ -49,6 +50,27 @@ class ServiceRegistryContract(Contract):
 
     contract_id = PUBLIC_ID
 
```

Comment:
Why all the not implemented methods?

Reply:
We implement the abstract methods but raise since we do not need them.